addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -4954,6 +4969,171 @@ public Action Command_CPR(int client, int args)
 	return Plugin_Handled;
 }
 
+public Action Command_PRinfo(int client, int args)
+{
+	if (!IsValidClient(client))
+		return Plugin_Handled;
+
+	switch(args){
+		//WORKS
+		//prinfo (in map zone) || (in bonus zone)
+		case 0:{
+			if(g_iClientInZone[client][2] == 0)
+				db_selectPRinfoUnknown(client, g_MapRank[client], 0, g_szSteamID[client]);
+			else
+				db_selectPRinfoUnknown(client, g_MapRankBonus[g_iClientInZone[client][2]][client], g_iClientInZone[client][2], g_szSteamID[client]);
+		}
+		//WORKS
+		//prinfo <mapname>
+		//prinfo <rank>
+		//prinfo <bonus number>
+		case 1:{
+			char arg1[128];
+			GetCmdArg(1, arg1, sizeof(arg1));
+
+			if (StrContains(arg1, "surf_", true) != -1)
+			{
+				db_viewPRinfoMapRank(client, g_szSteamID[client], arg1);
+			}
+			else if (arg1[0] == '@')
+			{
+				ReplaceString(arg1, sizeof(arg1), "@", "");
+				int rank = StringToInt(arg1);
+
+				//db_selectPRinfoUnknownWithMap(client, rank, g_szMapName, 0);
+
+				if(g_iClientInZone[client][2] == 0)
+					db_GetRankSteamID(client, g_szMapName, rank, 0);
+				else
+					db_GetRankSteamID(client, g_szMapName , rank, g_iClientInZone[client][2]);
+
+			}
+			else if (arg1[0] == 'b')
+			{
+				ReplaceString(arg1, sizeof(arg1), "b", "");
+				int bonus_number = StringToInt(arg1);
+
+				if (0 < bonus_number < MAXZONEGROUPS)
+					db_selectPRinfoUnknown(client, g_MapRankBonus[bonus_number][client], bonus_number, g_szSteamID[client]);
+				else
+					CPrintToChat(client, "%t", "InvalidBonusID", g_szChatPrefix, bonus_number);
```

Comment:
To touch a bit more on this [comment](https://github.com/surftimer/SurfTimer/pull/424/files#r877808261).

Here you are passing through 2 variables to the translation as that's how the translation made to be used and you'd have to pass it an `int` for it's second variable.
```
	"InvalidBonusID"
	{
		"#format"	"{1:s},{2:i}"
		"en"		"{1} Invalid bonus ID {2}"
	}
```

Reply:
from my testing when using clients laguange ("%T") i do need to pass the client index, when using("%t") that is not necessary :)